crates/uv/src/commands/python/install.rs

```diff
@@ -433,6 +438,23 @@ pub(crate) async fn install(
         })
     };
 
+    // For all satisfied installs, bytecode compile them now before any future
+    // early return.
+    if compile_bytecode {
+        let mut satisfied = satisfied.clone();
+        satisfied.sort();
```

Comment:
Is there a specific reason to sort inside here, vs. ensuring we're getting a fixed order from previous code?